Resources/Locale/en-US/_L5/traits/skills.ftl

```diff
@@ -0,0 +1,10 @@
+trait-category-skills = Skills
```

Comment:
These are good for now, but I would like to see them merged into unified Basic and Advanced Medical Training traits in preparation for departmental skills.

Reply:
Will tackle this and the mutually exclusive bits in a follow-up branch on traits. #241 and #242